05/app.js

```diff
@@ -10,26 +10,52 @@ const stats = {
 /* tutaj umieść swój kod */
 
 
+
+const allP = document.querySelectorAll('p');
+console.log(allP)
+
+
+const calculateClick = function (e) {
+
+
+    e.preventDefault();
+
+    const id = this.dataset.id;
+
```

Comment:
Jeśli chodzi o zliczanie dla `a` to sprawdź co zawiera `e.target` jak klikasz w `<p>` a co jak w `<a>`